doc/source/io.rst

```diff
@@ -3150,9 +3139,7 @@ The default is to 'infer
 msgpack
 -------
 
-.. versionadded:: 0.13.0
-
-Starting in 0.13.0, pandas is supporting the ``msgpack`` format for
+pandas supports the ``msgpack`` format for
```

Comment:
is this just underlyined in the diff?

Reply:
There no underline here: https://github.com/topper-123/pandas/blob/15e94c4aad7065db86e713e219aed94b6a80d800/doc/source/io.rst. a bit weird, but my  guess is that it's ok

Can't see anything stand out locally either.